extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/error/FrameFieldWriterFault.java

```diff
@@ -49,7 +51,7 @@ public FrameFieldWriterFault(
     super(
         CODE,
         StringUtils.format(
-            "Error[%s] while writing frame for source[%s], rowNumber[%d] column[%s]",
+            "Error[%s] while writing a field for source[%s], rowNumber[%d], column[%s].",
```

Comment:
nit: Duplicate error message as `FrameFieldWriterException` , can extract the format string out. It will make the updates easier.

Reply:
Now propagate the RE's message directly to the Fault class.